guake/test_terminal.py

```diff
@@ -26,14 +26,9 @@
 import os
 
 # pylint: disable=wrong-import-position
-import gi
-gi.require_version('Gtk', '3.0')
-gi.require_version('Notify', '0.7')
-gi.require_version('Vte', '2.91')
-
-from gi.repository import GLib
-from gi.repository import Gtk
-from gi.repository import Vte
+from guake import gi
+
+from gi.repository import GLib, Gtk, Vte
```

Comment:
see [guake/__init__.py](https://github.com/Guake/guake/pull/880/files#diff-6ca81edf8e502556eebff7aa46fbc5c1)
In my opinion, it`s better to keep this version bindings in a single place.